### ISSUE TITLE
USER ISSUE: World Destroyed 

### ISSUE THREAD

**Issue by EliteKampfRatte** [StrangeLoopGames/EcoIssues#6631]
**Version:** 0.7.3.0 beta

**Steps to Reproduce:**
Impacted Meteor

**Expected behavior:**
Close current world

**Actual behavior:**
other worlds are the same world as the world who are destroyed

**Comment by UnknownViper** [comment]
I'm guessing you want to start a new game? In that case you have to generate a new world.
If you're running the server you need to delete the \Storage\ folder to get the server to generate a new world for you, otherwise it will be using your old world. Close the server, delete, start server again.
I'm not sure how it works if you're playing single player, but in any case you can go to where Eco is installed and find the \Storage\ folder within the \Eco_Data\Server\ folder and delete it, then start the game again.

**Comment by MarjieAVolk** [comment]
Okay, I've gotten a handful of reports of new game opening up an old save.  I had been thinking people were making new games with the same name as their old games.  But it should check and give you a message for that now.  So it's gotta be something else.

**Comment by metachronism** [comment]
sounds like a messaging problem?  When you make a new world in singleplayer, you will be given the same world unless you click "generate new" in the dialog.

**Comment by MarjieAVolk** [comment]
No, they are getting their existing world, with the things they already built in it, etc.  Not just the same terrain generation.
